requirements-dev.txt

```diff
@@ -1,5 +1,5 @@
 dash_html_components
-dash_renderer
+dash_renderer>=0.12.1
```

Comment:
Is this just for the React 16 change?

Reply:
Yes, otherwise can't use normal API to test React 16 version tests. Can shim something in for the tests to not update the version if you prefer.